pkg/udf/forward/forward.go

```diff
@@ -242,9 +243,12 @@ func (isdf *InterStepDataForward) forwardAChunk(ctx context.Context) {
 	for _, msg := range dataMessages {
 		msg.Watermark = time.Time(processorWM)
 		// emit message size metric
-		metrics.ReadBytesCount.With(map[string]string{metrics.LabelVertex: isdf.vertexName, metrics.LabelPipeline: isdf.pipelineName, metrics.LabelVertexType: string(dfv1.VertexTypeMapUDF), metrics.LabelVertexReplicaIndex: strconv.Itoa(int(isdf.vertexReplica)), metrics.LabelPartitionName: isdf.fromBufferPartition.GetName()}).Add(float64(len(msg.Payload)))
+
+		totalBytes += len(msg.Payload)
```

Comment:
@whynowy Here we are increasing the metric only for data messages. 
We should use this metric for all read messages right? Data  + ctrl

I realized in the UDF we have been updating it only for data messags, and other types have for all

Reply:
Good point!

Reply:
This metric is only for data messages, we should make it consistent. Same for message count.

Reply:
In that case for all other UDFs we should update this, we are publishing for both data + ctrl.